docs/changehistory/NextVersion.md

```diff
@@ -1 +1,13 @@
 # NextVersion <!-- omit from toc -->
+
+Table of contents:
+
+- [@itwin/components-react](#itwincomponents-react)
+  - [Additions](#additions)
+
+## @itwin/components-react
+
+### Additions
+
+- Added ability to opt in to using new editors system in `FilterBuilder` component. [#1288](https://github.com/iTwin/appui/pull/1288)
+- Added ability to render status message in `EditorRenderer`. [#1288](https://github.com/iTwin/appui/pull/1288)
```

Comment:
Maybe they could be a bit more detailed, e.g. provide a small code snippet showing what prop to use